test/test_linalg.py

```diff
@@ -19,6 +19,44 @@
 class TestLinalg(TestCase):
     exact_dtype = True
 
+    @dtypes(torch.float, torch.double)
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
```

Comment:
We now require NumPy to run the test suite so these checks are no longer needed.